.circleci/config.yml

```diff
@@ -43,7 +43,8 @@ jobs:
           name: E2E tests
           command: |
             DocID=$(curl -s --data-binary @data/drugcases.qvf http://localhost:9076/v1/apps/import | jq -r '.attributes.id')
-            npm run start:cov &
+            npm run start:build
+            npm run start:serve &
```

Comment:
I think that serve will build it anyway :-(
But maybe faster

Reply:
Something is still wrong. But I think I'm on the right track with this. It's still failing now with this approach. I'm looking at it...

Reply:
I had to switch server to start using `http-server`. The build is now done synchronously and after that we start an http server using the dist folder.